.github/libs/GithubUtils.ts

```diff
@@ -333,12 +392,9 @@ class GithubUtils {
 
     /**
      * Fetch all pull requests given a list of PR numbers.
-     *
-     * @param {Array<Number>} pullRequestNumbers
-     * @returns {Promise}
      */
-    static fetchAllPullRequests(pullRequestNumbers) {
-        const oldestPR = _.first(_.sortBy(pullRequestNumbers));
+    static fetchAllPullRequests(pullRequestNumbers: number[]): Promise<OctokitPR[] | void> {
+        const oldestPR = pullRequestNumbers.sort()[0];
```

Comment:
careful! default sort of an array of numbers in JS does not work in an intuitive way, at all...

<img width="434" alt="image" src="https://github.com/Expensify/App/assets/47436092/fbdb9eef-ccb3-403e-b5b2-dbfcfd97908c">

```suggestion
        const oldestPR = pullRequestNumbers.sort((a, b) => a - b)[0];
```

<img width="544" alt="image" src="https://github.com/Expensify/App/assets/47436092/19b122fd-be86-41e9-903f-160160c53c33">

Reply:
Good point, thanks!